### ISSUE TITLE
Dear Author, during training, I noticed that the learning rate is increasing. Was this done intentionally?

### ISSUE THREAD

**Issue by ZACP-DERMATOLOGY** [ChingHo97/FCOS-PyTorch-37.2AP#32]
Dear Author, during training, I noticed that the learning rate is increasing. Was this done intentionally?

<img width="1468" height="710" alt="Image" src="https://github.com/user-attachments/assets/a9b320fd-3df6-4e77-a0a8-5a57484df4ce" />
